### ISSUE TITLE
Anti Piracy support

### ISSUE THREAD

**Issue by luke161** [playgameservices/play-games-plugin-for-unity#1595]
Is there anyway to check for `LICENSE_CHECK_FAILED` and does the plugin support Google Play Games anti-piracy? From what I can tell it would just return as unsuccessful when authorising a user, but that doesn't let us distinguish between other reasons.

https://developers.google.com/games/services/android/antipiracy

Thanks!

**Comment by Tanek81** [comment]
+1 on this question/request.

**Comment by Jacuos** [comment]
Yeah, it's been more than a year and this quite important "game" feature is still not supported by the plugin :-(

**Comment by prodigga** [comment]
Bump - any movement on this ?

**Comment by kuoliangkwong** [comment]
It's 2022 and it's still not being implemented. I checked from the code and it seems like `CommonStatusCodes.LicenseCheckFailed` enum with error code `11` is not being catered. Instead, it returns the default `SignInStatus.Failed` when it gets the license check failed error code internally. 

Any plan on adding this feature?

**Comment by KirillKuzyk** [comment]
I developed a plugin that enables you to check the Google Play license with a few lines of code:
https://forum.unity.com/threads/google-play-license-check-protect-your-game-from-piracy-and-hacking.1311378/
